Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -1,105 +1,95 @@
 /*  Style applied to the form grid table's outer structure  */
-table.formGridTable
+.formGridTable
```

Comment:
With `:root`, `--formgrid-border` is now globally accessible, though. Not that it poses a problem in this case, I assume, but do we really want that? Or put differently, if we make it globally accessible with `:root`, why not put it into `Common.css` in the first place, then it is at least visible/obvious to everyone?

Reply:
@a-ctor thoughts?

Reply:
I think both solutions are acceptable. Having it in the same file makes development easier as you quickly know what relevant variables are available to you. On the other hand, it is also beneficial to a have all of the "public API" declared in the same file.

I personally prefer the variables in the relevant files because if you want to style a FormGrid, for example, you go to `FormGrid.css` and look at the top lines. In the `common.css` you have to pick the relevant styles out of 100 variable declarations.

Reply:
okay, thanks and I tend towards the same reason for placing them in the same file.

 @a-ctor, the other question is regarding the variable being available as "public" vs. just within the formgrid.
 Hmm... maybe also get @iam-mholle  opinon.

Reply:
👍🏼 for the same file, 👍🏼 for `.formGridTable` instead of `:root` - why make it globally useable if its specific to this control?

Reply:
If we do not want anyone to change a variable it might be better to put it on the control class. But all the classes that have a chance of being overriden should be on `:root` and not the form control class. Otherwise it makes overriding the variable more difficult than it needs to be.

I would prefer adding all variables to `:root` because this allows them to be overridden without much drama, but there is the problem of differentiating public from private variables. If we found a good way to fix this I would go with all vars on `:root`. Maybe commenting `public` and `private` regions is enough? Or a prefix for private variables?

```
:root
{
  /* public */
  --my-public-var: ... ;

  /* private */
  --my-private-var: ... ;
}
```

Reply:
Another solution would be to declare private variables in the control class but do so in a way that keeps the original value if set:

```
.my-control
{
  --my-private-var = var(--my-private-var, default if not set);
}
```

This way, the variable is not declared as public API but can be overriden if needed as if it was declared in `:root`

Reply:
Okay, I think I've now a good overview and we should move this discussion outside of the current scope. There's three four places where we have some variables (FormGrid, SingleView/MultiView, and BocList). A few of those variables are extensionpoints and aside from SingleView/MultiView we have them inside the css-class at this time. I would suggest using the css-class for now (@a-ctor that's RM-7923) and discuss this in a design session regarding further improvements. Otherwise, we might blow the scope.